### PR TITLE
Update catalan language

### PR DESCRIPTION
**Pull request by divendres**
Update catalan language

### CODE REVIEW

**Comment by thinkyhead** [comment]
Thank you! Did the line endings get changed, perchance? Files should be saved with UNIX line endings.

**Comment by divendres** [comment]
thinkyhead, i think it's because i used the github editor. I did a pull request before but not on this branch and someone asked me to do it again here.
Do I need to do change it again using another editor?

**Comment by thinkyhead** [comment]
I see the problem. Not line-endings, but you dropped the macro `_UxGT()` from all the strings.

File: Marlin/language_ca.h [hunk 1]
```diff
@@ -33,125 +33,227 @@
...
-#define MSG_DELTA_CALIBRATE_CENTER          _UxGT("Calibrate Center")
+#define WELCOME_MSG                         MACHINE_NAME " preparada."
+#define MSG_SD_INSERTED                     "Targeta detectada."
+#define MSG_SD_REMOVED                      "Targeta extreta."
+#define MSG_LCD_ENDSTOPS                    "Endstops"
+#define MSG_MAIN                            "Menú principal"
+#define MSG_AUTOSTART                       "Inici automatic"
+#define MSG_DISABLE_STEPPERS                "Desactiva motors"
+#define MSG_DEBUG_MENU                      "Menu de depuracio"
+#define MSG_PROGRESS_BAR_TEST      "Test barra progres")
```
Comment (thinkyhead) [inline Marlin/language_ca.h]: …then there are lines that are just broken, like this one.

File: Marlin/language_ca.h [hunk 2]
```diff
@@ -33,125 +33,227 @@
...
+#if LCD_HEIGHT >= 4
+  #define MSG_FILAMENT_CHANGE_INIT_1          "Esperant per"
+  #define MSG_FILAMENT_CHANGE_INIT_2          "iniciar el canvi"
+  #define MSG_FILAMENT_CHANGE_INIT_3          "de filament"
+  #define MSG_FILAMENT_CHANGE_UNLOAD_1        "Esperant per"
+  #define MSG_FILAMENT_CHANGE_UNLOAD_2        "treure filament"
+  #define MSG_FILAMENT_CHANGE_INSERT_1        "Poseu filament"
+  #define MSG_FILAMENT_CHANGE_INSERT_2        "i premeu el boto"
+  #define MSG_FILAMENT_CHANGE_INSERT_3        "per continuar..."
+  #define MSG_FILAMENT_CHANGE_HEAT_1          "Premeu boto per")
```
Comment (thinkyhead) [inline Marlin/language_ca.h]: …and these…

File: Marlin/language_ca.h [hunk 3]
```diff
@@ -33,125 +33,227 @@
...
+  #define MSG_FILAMENT_CHANGE_HEAT_2          "escalfar nozzle.")
+  #define MSG_FILAMENT_CHANGE_HEATING_1       "Escalfant nozzle")
+  #define MSG_FILAMENT_CHANGE_HEATING_2       "Espereu...")
```
Comment (thinkyhead) [inline Marlin/language_ca.h]: …and these…

**Comment by thinkyhead** [comment]
Please confirm [the patch](https://github.com/divendres/Marlin/issues/2) when you have a chance. Looking forward to merging these updates!

**Comment by divendres** [comment]
I removed all this _Ux8G because in other languages they don't use it.
Sorry for not seeing all this closing paretheses.

**Comment by thinkyhead** [comment]
Squashed for merge at #5943

**Pull request closed without merging**